procrastinate/sql/schema.sql

```diff
@@ -40,6 +45,7 @@ CREATE TABLE procrastinate_jobs (
     scheduled_at timestamp with time zone NULL,
     attempts integer DEFAULT 0 NOT NULL,
     abort_requested boolean DEFAULT false NOT NULL,
+    worker_id bigint REFERENCES procrastinate_workers(id) ON DELETE SET NULL,
```

Comment:
This might also need an index to help the join operation of the `get_stalled_jobs` query ( excluding `NULL`)